src/components/Dialog/Dialog.tsx

```diff
@@ -120,4 +166,33 @@ export class Dialog extends React.Component<IDialogProps, any> {
 
     return groupings;
   }
+
+  private _onDialogRef(ref: HTMLDivElement) {
+    if (ref) {
```

Comment:
Looks like the `onAnimationEnd` handler was added for React 15. Here's the [pull request](https://github.com/facebook/react/pull/6005).